core/src/main/java/com/sequenceiq/cloudbreak/repository/InstanceMetaDataRepository.java

```diff
@@ -46,6 +46,9 @@
     @Query("SELECT i FROM InstanceMetaData i WHERE i.instanceGroup.stack.id= :stackId")
     Set<InstanceMetaData> findAllInStack(@Param("stackId") Long stackId);
 
+    @Query("SELECT i FROM InstanceMetaData i WHERE i.instanceGroup.stack.id= :stackId")
```

Comment:
This query looks the same as the above. What is the difference other than naming?

Reply:
Ahh I see, different entity graph. Sorry.